internal/server/config.go

```diff
@@ -32,12 +32,13 @@ type Jobs struct {
 }
 
 type Config struct {
-	Port                   int              `mapstructure:"port" default:"8080"`
-	GRPC                   GRPCConfig       `mapstructure:"grpc"`
-	EncryptionSecretKeyKey string           `mapstructure:"encryption_secret_key"`
-	Notifier               notifiers.Config `mapstructure:"notifier"`
-	LogLevel               string           `mapstructure:"log_level" default:"info"`
-	DB                     store.Config     `mapstructure:"db"`
+	Port                   int                         `mapstructure:"port" default:"8080"`
+	GRPC                   GRPCConfig                  `mapstructure:"grpc"`
+	EncryptionSecretKeyKey string                      `mapstructure:"encryption_secret_key"`
+	Notifier               notifiers.Config            `mapstructure:"notifier"`
+	Notifiers              notifiers.ConfigMultiClient `mapstructure:"notifiers"`
```

Comment:
this could be a map of config directly
```suggestion
	Notifiers              map[string]notifiers.Config `mapstructure:"notifiers"`
```

or any `map[string]struct` with following fields:
```go
type x struct {
  Provider string // slack | lark
  Criteria string
  Messages domain.NotificationMessages
  
  // slack config, required if Provider == slack
  AccessToken string
  
  // lark config, required if Provider == lark
  ClientID string
  ClientSecret string
}
```

Reply:
ConfigMultiClient is actually map[String]Notifier. and Notifier is the struct that you mentioned.